tools/c-parser/README.md

```diff
@@ -63,6 +63,8 @@ They have to be provided in the order listed below.
              frame rules
 - `c_types`: only define the types in the C code, leave out function definitions
 - `c_defs`: only provide C definitions, leave out type definitions
+- `no_modifies`: do not generate "modifies" proofs (frame theorems about which
+  globals each function can change).
```

Comment:
brain parse trouble, maybe "frame theorems which indicate the globals each function can change"?